pkg/types/types.go

```diff
@@ -228,10 +229,25 @@ func (e *L3Endpoint) String() string {
 	}
 }
 
+// ProtoToString converts an IP protocol number to its name.
+// https://www.iana.org/assignments/protocol-numbers/protocol-numbers.xhtml
+// TODO: Support other protos
+func ProtoToString(proto uint16) string {
```

Comment:
#1943 introduces the same function ProtoString(). If #1943 is merged first, this should be refactored to avoid duplicate functions.

Reply:
Fixed. Thanks for the heads up.